internal/config/config.go

```diff
@@ -97,6 +112,22 @@ func Read() (HCCMConfiguration, error) {
 		errs = append(errs, err)
 	}
 
+	cfg.Robot.Username = os.Getenv(robotUserName)
+	cfg.Robot.Password = os.Getenv(robotPassword)
+	// Enable Robot Support when both credentials are set
+	cfg.Robot.Enabled = cfg.Robot.Username != "" && cfg.Robot.Password != ""
```

Comment:
Is it not better to have a flag to enable robot, and fail hard on missing credentials ?

Reply:
Nice idea, implemented in next push :)